saleor/dashboard/order/forms.py

```diff
@@ -251,19 +251,22 @@ class OrderNoteForm(forms.Form):
         label=pgettext_lazy('Order note', 'Note'), widget=forms.Textarea())
 
 
-class ManagePaymentForm(forms.Form):
+class BasePaymentForm(forms.Form):
+
     amount = forms.DecimalField(
         label=pgettext_lazy(
             'Payment management form (capture, refund, void)', 'Amount'),
         max_digits=settings.DEFAULT_MAX_DIGITS,
         decimal_places=settings.DEFAULT_DECIMAL_PLACES)
 
+    clean_status = tuple()
```

Comment:
As I understand this tuple holds statuses that are allowed to perform an action that the form performs, right? We could rename it to something like `precondition_statuses` to make it more clear. `clean_status` could be interpreted in different ways. 

Side note - maybe that's outside the scope of this PR but shouldn't this validation live outside of forms? That would make it easier to reuse it the API.

Reply:
I think we should use methods that are available in the Payment model, such as `can_capture`, `can_refund` etc. There should be only one place that defined whether a particular action can be performed.

Reply:
Indeed! I've refactored it and now payment's built-in methods are used for validation.